src/Controller/Component/ApiCacheComponent.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * BEdita, API-first content management framework
+ * Copyright 2021 Atlas Srl, ChannelWeb Srl, Chialab Srl
+ *
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * See LICENSE.LGPL or <http://gnu.org/licenses/lgpl-3.0.html> for more details.
+ */
+
+namespace BEdita\WebTools\Controller\Component;
+
+use BEdita\WebTools\ApiClientProvider;
+use Cake\Cache\Cache;
+use Cake\Controller\Component;
+
+/**
+ * Component to cache some GET API calls.
+ *
+ * An index is used in order to refresh this cache with an external script.
+ */
+class ApiCacheComponent extends Component
+{
+    /**
+     * Default config for this component.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'cache' => '_apicache_',
+    ];
+
+    /**
+     * Use 'default' as fallback if no cache configuration is found.
+     *
+     * @param array $config The configuration settings provided to this helper.
+     * @return void
+     */
+    public function initialize(array $config): void
+    {
+        if (Cache::getConfig($this->getConfig('cache')) === null) {
```

Comment:
We should first use `$this->setConfig('cache', $config['cache'])` if a different cache config is passed